src/common/fees.ts

```diff
@@ -49,39 +49,40 @@ export const makeFees = async (config: MakeFeesConfig): Promise<Fees> => {
   // The last time the fees were updated
   let timestamp = 0
 
-  const updateVendorFees = async (): Promise<void> => {
+  const updateEdgeFees = async (): Promise<void> => {
     if (Date.now() - timestamp <= currencyInfo.feeUpdateInterval) return
 
-    const vendorFees = await fetchFeesFromVendor({
-      earnComFeeInfoServer: currencyInfo.earnComFeeInfoServer,
-      mempoolSpaceFeeInfoServer: currencyInfo.mempoolSpaceFeeInfoServer,
-      io,
-      log
-    })
-    fees = { ...fees, ...vendorFees }
+    const edgeFees = await fetchFeesFromEdge({ currencyInfo, io, log })
+    fees = { ...fees, ...edgeFees }
     timestamp = Date.now()
 
     await cacheFees(memlet, fees)
   }
 
-  let vendorIntervalId: NodeJS.Timeout
+  let edgeIntervalId: NodeJS.Timeout
 
   return {
     async start(): Promise<void> {
+      const vendorFees = await fetchFeesFromVendor({
```

Comment:
fetch fees from edge,
fetch fees from vendors - if ANY of them return then continue